web/apps/labelstudio/src/pages/CreateProject/CreateProject.jsx

```diff
@@ -126,13 +128,29 @@ export const CreateProject = ({ onClose, redirect = true }) => {
     }),
     [name, description, config],
   );
+  const uploadSample = useCallback(
```

Comment:
I would suggest to adding the interaction with the API to a hook and using it the same way as line 109. Probably the hook should be located in `labelstudio/src/hooks`